### PR TITLE
New note about files without file extensions

### PR DESCRIPTION
**Pull request by jmadin**
The requested note about file extensions given the decision to add .txt to files from shell redirects in Pipes and Filters.

### CODE REVIEW

File: instructors.md [hunk 1]
```diff
@@ -123,6 +123,15 @@ as long as learners using Windows do not run into roadblocks such as:
     and then retroactively decide which are worth encapsulating
     for future re-use.
 
+*   If everything is going well, you can drive home the point that file 
+    extensions are essetially there to help computers (a human 
+    readers) understand file content and are not a requirement of files. 
+    This can be done in the Pipes and Filters section by showing that you
```
Comment (gvwilson) [inline instructors.md]: Please make "Pipes and Filters" a link to that topic.

File: instructors.md [hunk 2]
```diff
@@ -123,6 +123,15 @@ as long as learners using Windows do not run into roadblocks such as:
...
     for future re-use.
 
+*   If everything is going well, you can drive home the point that file 
+    extensions are essetially there to help computers (a human 
+    readers) understand file content and are not a requirement of files. 
+    This can be done in the Pipes and Filters section by showing that you 
+    can redirect standard output to a file without the .txt extension 
+    (e.g., lengths), and that the resulting file is still a perfectly usable text file. 
+    Make the point that if double-clicked in the GUI, the computer will 
+    probably ask you what you want to do.
```
Comment (gvwilson) [inline instructors.md]: Good note.

File: instructors.md [hunk 3]
```diff
@@ -123,6 +123,15 @@ as long as learners using Windows do not run into roadblocks such as:
     and then retroactively decide which are worth encapsulating
     for future re-use.
 
+*   If everything is going well, you can drive home the point that file 
+    extensions are essetially there to help computers (a human
```
Comment (ChristinaLK) [inline instructors.md]: ("and human..."
?

Comment (jmadin) [inline instructors.md]: oops. Should be "and".  I'll fix this and add the link for Pipes and Filters.

File: instructors.md (hunk 1 above)
Comment (wking) [inline instructors.md]: On Mon, Feb 16, 2015 at 06:46:35PM -0800, Greg Wilson wrote:
> > +*   If everything is going well, you can drive home the point that file 
> > +    extensions are essetially there to help computers (a human 

“essetially” → “essentially”.

**Pull request merged**